queries/omen/markets.ts

```diff
@@ -51,6 +50,7 @@ const getMarketQuery = gql`
       answerFinalizedTimestamp
       scaledLiquidityParameter
       scaledLiquidityMeasure
+      liquidityMeasure
```

Comment:
what is this field?

Reply:
This is liquidity but with not formatted to decimal number. Will delete it since it's not being used anymore.